mne/html_templates/report/footer.html.jinja

```diff
@@ -1,3 +1,11 @@
+{# Complete tags for curlylint #}
+{{ if 0 }}
+<html lang="{{ lang | safe }}">
+<body>
+<div>
+<div>
+{{ endif }}
```

Comment:
Added `curlylint`. It necessitated adding stuff like this, which I actually don't mind -- it helps us document a bit what HTML tags we have opened and/or will close later.